backend/src/types/types.d.ts

```diff
@@ -484,4 +490,8 @@ declare namespace MonkeyTypes {
     frontendForcedConfig?: Record<string, string[] | boolean[]>;
     frontendFunctions?: string[];
   }
+
+  interface PremiumProperties {
+    isActive: boolean;
+  }
```

Comment:
is there a spec for this feature?
Think it would make more sense to store a timestamp here, and provide utilities in app code to determine if a user's premium is active.

```suggestion
  interface PremiumInfo {
    expirationTimestamp: number;
  }
```

Reply:
Maybe even include start timestamp?

Reply:
perhaps even a `type` field if we want to support tiered memberships, but that would have a huge change on your overall implementation.